documentation/migrating/01-migrating.md

```diff
@@ -7,10 +7,10 @@ SvelteKit is largely backwards-compatible with Sapper. However, there are a numb
 - Remove `rollup.config.js` or `webpack.config.js` and replace with [`vite.config.js`](https://github.com/sveltejs/kit/blob/master/packages/create-svelte/template/vite.config.js)
 - Add a [`svelte.config.cjs` ](https://github.com/sveltejs/kit/blob/master/packages/create-svelte/template/svelte.config.cjs) with the adapter of your choice
 - Update your `template.html` to [`app.html`](https://github.com/sveltejs/kit/blob/master/packages/create-svelte/template/src/app.html)
-	- It should have a `<div id="svelte">` (or `id` matching `target` in `svelte.config.cjs`)
+  - It should have a `<div id="svelte">` (or `id` matching `target` in `svelte.config.cjs`)
 - The error and layout pages should prefixed by `$` instead of `_`
 - Replace imports from `@sapper/app` with imports from `$app/navigation` and `$app/stores` and update APIs accordingly
-- If you have `src/node_modules/components`, move it to `src/components` and update the import path to `$components`
+- If you have `src/node_modules/components`, move it to `src/lib` and update the import path to `$lib`
```

Comment:
It didn't change specifically in this PR, but I think this could be a bit more descriptive. _Anything_ in `src/node_modules` is now a problem, and needs to be moved.

Reply:
good catch